src/LibPQ.jl

```diff
@@ -3,7 +3,8 @@ module LibPQ
 export Connection, Result
 export status, reset!, execute, clear
 
-using DocStringExtensions, DataStreams, Nulls, NullableArrays
+using DocStringExtensions, DataStreams, Nulls, NullableArrays,
+    IteratorInterfaceExtensions, TableTraits, DataValues, NamedTuples
```

Comment:
Some packages also add an ``import IterableTables`` here to make sure the iterable tables integration for ``DataFrame`` is always loaded.